bee-node/src/plugins/dashboard/config.rs

```diff
@@ -74,16 +76,44 @@ impl DashboardConfigBuilder {
     }
 
     pub fn finish(self) -> DashboardConfig {
+        let multi_addr = self
+            .bind_addresses
+            .unwrap_or_else(|| DEFAULT_BIND_ADDRESSES.parse().unwrap());
+        let address = multi_addr
+            .iter()
+            .find_map(|x| match x {
+                Protocol::Dns(std::borrow::Cow::Borrowed("localhost")) => Some(
```

Comment:
Should we really limit ourselves to "localhost"? Why would we not allow any domain name for the sake of supporting anybody's (maybe sometimes unusual) configuration?

Reply:
You are right. I have never seen anyone use anything else, but then I should not infer from me to others 👍